app/views/sir-trevor/blocks/_instagram_block.html.erb

```diff
@@ -1,6 +1,10 @@
 <div class="content-block instagram-block">
   <%=
-    if block['instagram_embed_code'] && block['instagram_embed_code'].html_safe.match(/\<blockquote (.*)\<\/blockquote>(\s+)\<script async defer src\=\"\/\/platform\.instagram\.com\/en_US\/embeds\.js\"\>\<\/script\>/)
+    instagram_block_patterns = [
+      /<blockquote (.*)<\/blockquote>(\s+)<script async defer src="\/\/platform.instagram.com\/en_US\/embeds.js"><\/script>/,
+      /<blockquote (.*)<\/blockquote>(\s+)<script async defer src="www.instagram.com\/embed.js"><\/script>/
+    ]
+    if block['instagram_embed_code'] && instagram_block_patterns.any? { |pattern| pattern.match(block['instagram_embed_code'].html_safe }
```

Comment:
Hey Laco, I might be blind but seems like this line is missing `)` :-)

Reply:
https://xkcd.com/859/

Reply:
Wow, thanks for checking Juro :) will code-check and test this before merging

we say hi from Represent :)